### ISSUE TITLE
UE5 says the Gideon Blueprint anim is corrupted

### ISSUE THREAD

**Issue by TongJiTony** [tomlooman/ActionRoguelike#9]
I downloaded the ExampleContent and ParagonGideon file and put them in the instucted position, but sadly found that the Gideon had no color and the blueprint could not be found.
When I clicked the blueprint in menu directly, I was told it was corrupted. 
-files were from master
-using ue5

**Comment by TongJiTony** [comment]
> I downloaded the ExampleContent and ParagonGideon file and put them in the instucted position, but sadly found that the Gideon had no color and the blueprint could not be found. When I clicked the blueprint in menu directly, I was told it was corrupted. -files were from master -using ue5

I have solved this by searching the Gideon in the laucher's store and installing them in the project

**Comment by bulabulabula818** [comment]
-using ue4.27.2
i meet the same problem
the Gideon's model is Ok but i can't load the animation blueprint and color

**Comment by TongJiTony** [comment]
Yes, and you could just download the content in the launcher's store directly and it would be bound to your project

**Comment by bulabulabula818** [comment]
Gideon's package about 2GB ......

**Comment by Eric5528** [comment]
I have meet this problem too. Don't try to copy the animation blueprint in your own created ones ,just download the Gideon on store.
